modules/notebook-images-for-data-scientists.adoc

```diff
@@ -12,7 +12,7 @@ Notebook images are upgraded quarterly to ensure that you are working with the l
 endif::[]
 
 ifndef::upstream[]
-Notebook images are supported for a minimum of one year. Major updates to pre-configured notebook images occur approximately every six months. Therefore, two supported notebook images are typically available at any given time. You can use this support period to update your code to use components from the latest available notebook image.
+Notebook images are supported for a minimum of one year. Major updates to pre-configured notebook images occur approximately every six months. Therefore, multiple supported notebook images are typically available at any given time. You can use this support period to update your code to use components from the latest available notebook image.
```

Comment:
In options-for-notebook-server-environments, you've updated 'approximately' to 'about' and 'notebook images' to 'notebook image versions'. Do we want to do that here as well?

Reply:
I'll update it here also.

Reply:
Updated.